src/components/BottomSheet/BottomSheetContent/BottomSheetContent.tsx

```diff
@@ -0,0 +1,12 @@
+import {View, Text} from 'react-native'
+import React from 'react'
+
+const BottomSheetContent = () => {
```

Comment:
Do we need this one? If not please remove it

Reply:
yes, we do